dali/test/python/test_external_source_parallel_custom_serialization.py

```diff
@@ -275,12 +273,12 @@ def _build_and_compare_pipelines_epochs(epochs_num, batch_size, parallel_pipelin
             serial_pipeline.reset()
 
 
-def _create_and_compare_simple_pipelines(cb, py_callback_pickler, batch_size, py_num_workers=2, py_start_method="spawn"):
-    parallel_pipeline = create_simple_pipeline(
-        cb, py_callback_pickler, batch_size=batch_size, py_num_workers=py_num_workers,
-        py_start_method=py_start_method, parallel=True)
-    serial_pipeline = create_simple_pipeline(
-        cb, None, batch_size=batch_size, parallel=False)
+def _create_and_compare_simple_pipelines(cb, py_callback_pickler, batch_size, py_num_workers=2,
+                                         py_start_method="spawn"):
+    parallel_pipeline = create_simple_pipeline(cb, py_callback_pickler, batch_size=batch_size,
+                                               py_num_workers=py_num_workers,
+                                               py_start_method=py_start_method, parallel=True)
```

Comment:
At this point, one keyword argument per line seems a like a better idea.
```suggestion
    parallel_pipeline = create_simple_pipeline(cb, py_callback_pickler,
                                               batch_size=batch_size,
                                               py_num_workers=py_num_workers,
                                               py_start_method=py_start_method,
                                               parallel=True)
```

Reply:
Done